### PR TITLE
br: synchronize the usage tips of the latest version to 5.4

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://internal.pingcap.net/jira/browse/DOCS-469

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -121,15 +121,13 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
-- BR 备份最好串行执行。不同备份任务并行会导致备份性能降低，同时也会影响在线业务。
-- BR 恢复最好串行执行。不同恢复任务并行会导致 Region 冲突增多，恢复的性能降低。
-- 推荐在 `-s` 指定的备份路径上挂载一个共享存储，例如 NFS。这样能方便收集和管理备份文件。
-- 在使用共享存储时，推荐使用高吞吐的存储硬件，因为存储的吞吐会限制备份或恢复的速度。
-- BR 默认会分别在备份、恢复完成后，进行一轮数据校验，将文本数据同集群数据比较，来保证正确性。在迁移场景下，为了减少迁移耗时，推荐备份时关闭校验(`--checksum=false`)，只在恢复时开启校验。
+下面是使用 BR 进行备份恢复的一些建议：
+
+- 推荐在业务低峰时执行备份操作，这样能最大程度地减少对业务的影响；
+- BR 只支持恢复数据到新集群，会尽可能多的使用恢复集群的资源。不推荐向正在提供服务的生产集群执行恢复，恢复期间会对业务产生不可避免的影响；
+- 不推荐多个 BR 备份和恢复任务并行运行。不同的任务并行，不仅会导致备份或恢复的性能降低，影响在线业务；还会因为任务之间缺少协调机制造成任务失败，甚至对集群的状态产生影响；
```
Comment (Oreoxmt) [inline br/backup-and-restore-tool.md:128]:
```suggestion
- 不推荐多个 BR 备份和恢复任务并行运行。不同的任务并行，不仅会导致备份或恢复的性能降低，影响在线业务，还会因为任务之间缺少协调机制造成任务失败，甚至对集群的状态产生影响；
```

**Comment by IANTHEREAL** [comment]
LGTM

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -121,15 +121,13 @@ BR 内置版本会在执行备份和恢复操作前，对 TiDB 集群版本和
...
-- BR 支持在不同拓扑的集群上执行恢复，但恢复期间对在线业务影响很大，建议低峰期或者限速 (`rate-limit`) 执行恢复。
-- BR 备份最好串行执行。不同备份任务并行会导致备份性能降低，同时也会影响在线业务。
-- BR 恢复最好串行执行。不同恢复任务并行会导致 Region 冲突增多，恢复的性能降低。
-- 推荐在 `-s` 指定的备份路径上挂载一个共享存储，例如 NFS。这样能方便收集和管理备份文件。
-- 在使用共享存储时，推荐使用高吞吐的存储硬件，因为存储的吞吐会限制备份或恢复的速度。
-- BR 默认会分别在备份、恢复完成后，进行一轮数据校验，将文本数据同集群数据比较，来保证正确性。在迁移场景下，为了减少迁移耗时，推荐备份时关闭校验(`--checksum=false`)，只在恢复时开启校验。
+下面是使用 BR 进行备份恢复的一些建议：
+
+- 推荐在业务低峰时执行备份操作，这样能最大程度地减少对业务的影响；
+- BR 只支持恢复数据到新集群，会尽可能多地使用恢复集群的资源。不推荐向正在提供服务的生产集群执行恢复，恢复期间会对业务产生不可避免的影响；
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply br/backup-and-restore-tool.md:127]: > 这句的逻辑我没太理解，“新集群”为什么会是提供服务的生产集群，尚未恢复完成的集群能够提供服务吗

这句话的重点在于 BR 恢复会占用很多资源，因此推荐在新集群（空集群）进行，而不是在正在提供服务的生产集群上进行。奥琳觉得这句话怎么修改会更好地表达这层意思呢？另外，玉龙老师提议在恢复的对象加上离线集群，即 `BR 只支持恢复数据到新集群和离线集群`。

Comment (shichun-0415) [inline br/backup-and-restore-tool.md:127]:
```suggestion
- BR 恢复数据时会尽可能多地占用恢复集群的资源，因此推荐恢复数据到新集群或离线集群。应避免向正在提供服务的生产集群执行恢复，否则，恢复期间会对业务产生不可避免的影响；
```

Comment (shichun-0415) [reply br/backup-and-restore-tool.md:127]: @Oreoxmt PTAL

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**